strings/salesman_replacement.json

```diff
@@ -0,0 +1,49 @@
+{
+
+    "salesman": {
+        "shuttle": "cungadero",
+        "clam": "pipis",
+        "clams": "pipis",
+        "egg": "pipis",
+        "eggs": "pipis",
+        "love": "hyperlink blocked",
+        "captain": "big shot",
+        "dollar": "kromer",
+        "dollars": "kromer",
+        "credit": "kromer",
+        "credits": "kromer",
+        "basement": "deep abyss",
+        "mech": "workout-ready body",
+        "exosuit": "workout-ready body",
+        "phazon": "NEO",
+        "bolts": "high quality encryption",
+        "jester": "clown around town",
```

Comment:
just replace clown, please

Reply:
cant do that because otherwise its recursive, i used to have clown be replaced but it didnt work out